doc/installing.rst

```diff
@@ -31,12 +31,18 @@ download page.
 System Requirements
 ----------------------------------
 
-- Windows 10+
+- Windows 10 minimum version 1709 [SAC] 
```

Comment:
@allexzander 
added the win8.1 line
to go with just 8.1+ is a bad idea because that includes 1511 again 😞 

remember: if nextcloud follows the m$ sac way, you have to always update
the requirements according to m$ ‼️ 

@githubdco
it is out of my reach to resolve the two remaining missing singoffs 🤦🏻‍♂️